DDF.xml

```diff
@@ -219,7 +219,7 @@ These LwM2M Object Resources MUST only be changed by a LwM2M Bootstrap-Server or
         <DDF>LWM2M_LOCKWIPE-v1_0_2.xml</DDF>
         <Vorto></Vorto>
         <DDFLink>1</DDFLink>
-        <TS>http://www.openmobilealliance.org/release/LWM2M_LOCKWIPE/V1_0_1-20181003-A/OMA-TS-LWM2M_LockWipe-V1_0-20180615-A.pdf</TS>
+        <TS>http://www.openmobilealliance.org/release/LWM2M_LOCKWIPE/V1_0_2-20200616-A/OMA-TS-LWM2M_LockWipe-V1_0_1-20200616-A.pdf</TS>
```

Comment:
why is v1.0.2, it should be version v1.0.1, isn't?

Reply:
`V1_0_2-20200616-A` is the correct file name. 
The TS has been updated to `v1_0_1` to align with the changes but the file reference is aligned with the latest ERP/RRP version, which for this Enabler is v1.0.2